net-snmp/run.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+(
+    . /etc/sysconfig/snmpd
+    if test "$WAIT_FOR_CONFIGURATION" == 1; then
+        systemd-notify --ready
+        exec sleep 9999d
```

Comment:
well that is just one example of the settings required, for example the system could require more users or any kind of setting.  I think the best way to handle this is to have some sort of thrown away runonce container here which is how I'd expect it to be handled in the Docker case as well: `docker run --rm -v /etc... -v /var/lib/net-snmp net-snmp-create-v3-user -X PASSWORD EXPECTED_USER`